force-app/main/default/triggers/RejectDuplicateFavorite.trigger

```diff
@@ -1,11 +1,14 @@
-trigger RejectDuplicateFavorite on Favorite__c (before insert) {
-    
-    // NOTE: this trigger needs to be bulkified
-    
-    Favorite__c favorite = Trigger.New[0];
-    List<Favorite__c> dupes = [Select Id FROM Favorite__C WHERE Property__c = :favorite.Property__c AND User__c = :favorite.User__c];
-    if (!dupes.isEmpty()) {
-        favorite.addError('duplicate');
-    }
+trigger RejectDuplicateFavorite on Favorite__c(before insert) {
+	// NOTE: this trigger needs to be bulkified
```

Comment:
let's bulkify it (comment was added on previous pr). And as we talked before RejectDuplicateFavourite name isn't good for triggers, as it should be some common like FavouriteTrigger